host/bloodview/src/data-cal.c

```diff
@@ -290,15 +357,17 @@ void *data_cal_init(
 		return NULL;
 	}
 
-	ctx->channel = data_cal__init_channel_array(channels);
+	unsigned channel_count = __builtin_popcount(channel_mask);
```

Comment:
For the bloodview code, prefer to use `util_bit_count()` than something GCC-specific.

Reply:
+1